contrib/net/http/http.go

```diff
@@ -10,8 +10,8 @@ import (
 	"net/http"
 
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
```

Comment:
nit: these imports can be grouped I believe

Reply:
Fixed